networking/synthetic/stac/tuned.conf

```diff
@@ -1,25 +1,14 @@
 # tuned configuration
 #
-
 [main]
-include=network-latency
+include=cpu-partitioning
 
-[cpu]
-force_latency=1
+[variables]
+# User is responsible for updating variables.conf with variable content such as isolated_cores=X-Y
+include=/etc/tuned/stac/cpu-partitioning-variables.conf
 
 [bootloader]
-# cmdline = YOUR_ADDITIONAL_KERNEL_PARAMETERS
-#cmdline = audit=0 selinux=0 isolcpus=1-15 nohz_full=1-15
-#cmdline = audit=0 selinux=0 isolcpus=1-15 nosoftlockup mce=ignore_ce idle=poll
-cmdline = audit=0 selinux=0 isolcpus=1-15 nosoftlockup mce=ignore_ce
+cmdline = audit=0 selinux=0 isolcpus=${isolated_cores} nosoftlockup mce=ignore_ce skew_tick=1
```

Comment:
what is unique about this bootloader line that we have to carry it?  isnt it the same as the one from cpu-partitioning?

Reply:
I advised with Joe about it and he suggested to keep it.
would you like to ping him about it ?!

Reply:
keep it, absolutely. but if it is no different than the one that cpu-partitioning would create, then why are you also carrying it in the stac tuned profile?

Reply:
the diffrence is about the cores. so now we inject the right one